manage-data/ingest/ingesting-data-from-applications/ingest-logs-from-nodejs-web-application-using-filebeat.md

```diff
@@ -20,514 +22,443 @@ This guide demonstrates how to ingest logs from a Node.js web application and de
+1. [Login to Kibana](../../../deploy-manage/deploy/elastic-cloud/access-kibana.md).
+2. Open the {{kib}} main menu, under **Observability** select **Discover**.
+3. In the **Data view** dropdown, select **`filebeat-*`**.
 
-    ![A screen capture of a donut chart divided into three sections](/manage-data/images/cloud-ec-node-logs-donut.png "")
+You should see the ingested events.
 
-4. Select **Save and return** to add this visualization to your dashboard.
-5. Select **Save** and add a title to save your new dashboard.
+::::
+
+:::::
 
-You now have a Kibana dashboard with three visualizations: a stacked bar chart showing the frequency of each HTTP request method over time, another stacked bar chart showing the frequency of various HTTP *from* headers over time, and a donut chart showing the relative frequency of each HTTP request method type.
 
-You can add titles to the visualizations, resize and position them as you like, and then save your changes.
+## Create log visualizations in Kibana [ec-node-logs-view-kibana]
 
-**View log data updates in real time**
+Now you can explore and visualize your ingested log data in Kibana to gain insights from your Node.js application.
 
-1. Select **Refresh** on the Kibana dashboard. Since the application `webrequests.js` continues to run and send HTTP requests to the Node.js server, `webserver.js` continues to generate log data, and your Kibana visualizations update with that data with each page refresh.
+To visualize your log data, refer to the [Kibana documentation on creating visualizations](../../../explore-analyze/visualize.md). This ensures you’re using the latest interface for your deployment.
 
-    ![A screen capture of the completed Kibana dashboard](/manage-data/images/cloud-ec-node-logs-final-dashboard.png "")
+## Clean up
 
-2. As your final step, remember to stop Filebeat, the Node.js web server, and the client. Enter *CTRL + C* in the terminal window for each application to stop them.
+Stop all running processes by pressing `Ctrl + C` in each terminal window.
 
-You now know how to monitor log files from a Node.js web application, deliver the log event data securely into an {{ech}} or {{ece}} deployment, and then visualize the results in Kibana in real time. Consult the [Filebeat documentation](beats://reference/filebeat/index.md) to learn more about the ingestion and processing options available for your data. You can also explore our [documentation](../../../manage-data/ingest.md) to learn all about ingesting data.
+You can delete temporary files such as the `logs/` directory and the node_modules folder when finished.
 
+You now know how to monitor log files from a Node.js web application, deliver the log event data securely into an {{ech}} or {{ece}} deployment, and then visualize the results in Kibana in real time. Consult the [Filebeat documentation](beats://reference/filebeat/index.md) to learn more about the ingestion and processing options available for your data.
```

Comment:
> You now know how to monitor (...)

Sounds like a tongue twist to me 😅. I know it was already there, but I believe it can use some improvement. What about:
```suggestion
You have learned how to monitor log files from a Node.js web application, deliver the log event data securely into an {{ech}} or {{ece}} deployment, and then visualize the results in Kibana in real time. Consult the [Filebeat documentation](beats://reference/filebeat/index.md) to learn more about the ingestion and processing options available for your data.
```